src/utils/chain.ts

```diff
@@ -13,22 +13,48 @@
  */
 
 import cli from 'cli-ux';
-import { getClient } from '../client';
+import { getAPIClient } from '../client';
```

Comment:
Please run `npm i -S @liskhq/lisk-chain` and make the following changes to replace `Record<string, any>` with `Record<string, unknown>` below.

```suggestion
import { Block, BlockHeader } from '@liskhq/lisk-chain';
import { getAPIClient } from '../client';
```